src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -478,4 +478,12 @@ protected function isHttpException(Exception $e)
     {
         return $e instanceof HttpException;
     }
+
+    /**
+     * @return \Illuminate\Config\Repository|mixed
+     */
+    private function isDebugMode()
```

Comment:
this method should be declared like protected

Reply:
I`m not sure, that we should do change it now.